packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/hooks.ts

```diff
@@ -1,13 +1,12 @@
-import { useStocktakeFields } from './../../../api/hooks';
-import { StocktakeLineFragment } from './../../../api/operations.generated';
 import React, { useEffect, Dispatch, SetStateAction } from 'react';
+import { RecordPatch, generateUUID } from '@openmsupply-client/common';
```

Comment:
For stocktakes there wasn't too much to change, but I did end up touching the `useStockLines` hook also to return a `StockLineFragment`